Corona-Warn-App/src/test/java/de/rki/coronawarnapp/nearby/windows/entities/cases/TestCase.kt

```diff
@@ -6,9 +6,9 @@ data class TestCase(
     @SerializedName("description")
     val description: String,
     @SerializedName("expAgeOfMostRecentDateWithHighRisk")
-    val expAgeOfMostRecentDateWithHighRisk: Long?,
+    val expAgeOfMostRecentDateWithHighRiskInDays: Long?,
```

Comment:
minor: if it is days, Int would suffice, i believe?

Reply:
I agree, doesn't matter though for the test.